resources/langs/nheko_de.ts

```diff
@@ -139,58 +139,60 @@
     <message>
         <location filename="../../src/dialogs/PreviewUploadOverlay.cc" line="41"/>
         <source>Upload</source>
-        <translation type="unfinished"></translation>
+        <translation>Hochladen</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/PreviewUploadOverlay.cc" line="42"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Abbrechen</translation>
     </message>
     <message>
         <location filename="../../src/dialogs/PreviewUploadOverlay.cc" line="114"/>
         <source>Media type: %1
 Media size: %2
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Medien Typ: %1
+Medien Größe: %2
```

Comment:
I would prefer to connect "Medien {Typ,Größe}" with "-"

Reply:
👍